apps/demo/src/app/drivers/shopify.module.ts

```diff
@@ -1,58 +1,46 @@
-import { HttpHeaders } from '@angular/common/http';
 import { NgModule } from '@angular/core';
+import { InMemoryCache } from '@apollo/client/core';
 import {
-  InMemoryCache,
-  ApolloLink,
-} from '@apollo/client/core';
-import { onError } from '@apollo/client/link/error';
-import { Apollo } from 'apollo-angular';
-import { HttpLink } from 'apollo-angular/http';
+  Apollo,
+  provideApollo,
+} from 'apollo-angular';
 
+
+import { DaffAuthorizeNetInMemoryDriverModule } from '@daffodil/authorizenet/driver/in-memory';
 import { DaffCartInMemoryDriverModule } from '@daffodil/cart/driver/in-memory';
+import { DaffCategoryInMemoryDriverModule } from '@daffodil/category/driver/in-memory';
+import { createApolloConfig } from '@daffodil/driver/shopify';
+import { DaffInMemoryDriverModule } from '@daffodil/driver/in-memory';
+import { DaffNavigationInMemoryDriverModule } from '@daffodil/navigation/driver/in-memory';
 import { DaffNewsletterInMemoryDriverModule } from '@daffodil/newsletter/driver/in-memory';
 import { DaffProductShopifyDriverModule } from '@daffodil/product/driver/shopify';
+import { shopifyDriverConfig } from '@daffodil/product/driver/shopify';
+import {
+  DaffDefaultProductFactory,
+  provideDaffProductExtraFactoryTypes,
+} from '@daffodil/product/testing';
+
 
 import { environment } from '../../environments/environment';
 import { ShopifyEnviromentDriverConfiguration } from '../../environments/environment.interface';
 
-const cache = new InMemoryCache();
-
 @NgModule({
   imports: [
-    //Shopify
+    DaffInMemoryDriverModule.forRoot(),
     DaffProductShopifyDriverModule.forRoot(),
     DaffCartInMemoryDriverModule.forRoot(),
     DaffNewsletterInMemoryDriverModule.forRoot(),
+    DaffCategoryInMemoryDriverModule.forRoot(),
+    DaffNavigationInMemoryDriverModule.forRoot(),
+    DaffAuthorizeNetInMemoryDriverModule.forRoot(),
+  ],
+  providers: [
+    provideDaffProductExtraFactoryTypes(DaffDefaultProductFactory),
+    provideApollo(
+      () => createApolloConfig(shopifyDriverConfig(
```

Comment:
This provider and subsequent functions are pretty uncomfortable to read, can you adjust this to look like:

```ts
provideShopifyApolloDriver(domain, accessToken)
```